sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/implementation/AzureCommunicationCallingServerServiceImplBuilder.java

```diff
@@ -190,7 +190,7 @@ public AzureCommunicationCallingServerServiceImplBuilder addPolicy(HttpPipelineP
      */
     public AzureCommunicationCallingServerServiceImpl buildClient() {
         if (apiVersion == null) {
-            this.apiVersion = "2021-04-15-preview1";
+            this.apiVersion = "2021-03-28-preview0";
```

Comment:
Why are we going back in version?

Reply:
I changed the time to the date C# PR sent out.